model/src/test_manager/status.rs

```diff
@@ -198,3 +245,22 @@ impl From<&Crd> for Vec<ResultRow> {
         results
     }
 }
+
+struct AdditionalColumn {
+    header: String,
+    value: fn(&Crd) -> Option<String>,
+}
+
+impl std::fmt::Debug for AdditionalColumn {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("AdditionalColumn")
+            .field("header", &self.header)
+            .finish()
+    }
+}
+
+#[test]
+fn test_new_columns() {
+    // Add a new column to the status with the crd's name.
+    StatusSnapshot::new(None, Default::default()).new_column("extra column", |crd| crd.name());
+}
```

Comment:
This test would never fail so I'm not sure if this is useful. If we want to show an example for adding a new custom column, we can probably have it as a docstring for `StatusSnapshot`.

Reply:
It would be tough to create an `assert` with the Tabled lib.